### ISSUE TITLE
[Fixed in GH-6052] Language and Currency dropdowns in header provide no visual focus feedback

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#4146]
You can control these dropdowns with arrows when focused, but there is no visual feedback that tells you that you are focused on the dropdown elements.

![image](https://user-images.githubusercontent.com/11735817/63434639-c0439780-c425-11e9-9538-65d6f3411610.png)

**Comment by KateChuen** [comment]
Still an issue on all 3 browsers.

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 3 00 47 PM](https://user-images.githubusercontent.com/15113219/73570630-af6a6980-443a-11ea-93a0-62d56a2d59e5.png)
![Screen Shot 2020-01-31 at 3 00 23 PM](https://user-images.githubusercontent.com/15113219/73570632-af6a6980-443a-11ea-8d83-c7a94f0ab1e5.png)
![Screen Shot 2020-01-31 at 2 59 55 PM](https://user-images.githubusercontent.com/15113219/73570633-af6a6980-443a-11ea-81a4-e6868210737a.png)

**Comment by KateChuen** [comment]
Tested on dev3
- Chrome 
- Firefox 
- Safari
